### PR TITLE
Add documentation for multiuser configuration

### PR DESCRIPTION
**Pull request by PaulTaykalo**
🔑 Documentation for multiuser team configuration

This is an example how Appfile can be configured in case if your team is big enough and multiple users can submit application to the AppStore

### CODE REVIEW

**Comment by KrauseFx** [comment]
Hey @PaulTaykalo, this is great, thanks for your PR. You could also mention the approach of using files that are in gitignore. Basically importing or reading the files if they exist

Appfile
```ruby
if File.exist?("apple_id.txt")
  apple_id File.read("apple_id.txt")
else
  apple_id "default@email.com"
end
```
.gitignore
```
apple_id.txt
```

**Comment by KrauseFx** [comment]
Could you drop me a short email, I'd love to invite you to the Slack group

File: fastlane/docs/Appfile.md [hunk 1]
```diff
@@ -63,3 +63,25 @@ If you want to access those values from within your `Fastfile` use
...
+In big teams instead of specifying one email for `apple_id`, you can get `apple_id` from environment variable instead. 
+Doing this will allow different developers to specify their own apple ids, and won't affect others
+In case if this enviroment variable won't be setup - then it will be asked by fastlane on any command that requires apple_id
+
+```ruby
+# without default fallback user will be asked to input apple_id
+apple_id ENV["MY_APP_NAME_APPLE_ID"] 
+
+# In case if environment variable isn't set up, fallback email will be used
+apple_id ENV["MY_APP_NAME_APPLE_ID"] || "defaultintegratoremail@gmail.com"
```
Comment (KrauseFx) [inline fastlane/docs/Appfile.md]: Let's use a nicer looking example email address 👍

**Comment by PaulTaykalo** [comment]
@KrauseFx something like this

File: fastlane/docs/Appfile.md [hunk 2]
```diff
@@ -63,3 +63,45 @@ If you want to access those values from within your `Fastfile` use
...
+export MY_APP_NAME_APPLE_ID="another_user@gmail.com"
+```
+
+#### Using file check and .gitignore
+
+As an another option, you can use use file checking and .gitignore combination:
+```ruby
+# Appfile
+if File.exist?("apple_id.txt")
+  apple_id File.read("apple_id.txt")
```
Comment (KrauseFx) [inline fastlane/docs/Appfile.md]: Let's use
```
apple_id File.read("apple_id.txt").strip
```

Comment (KrauseFx) [inline fastlane/docs/Appfile.md]: This will make sure no new-line is there

**Comment by KrauseFx** [comment]
Oh, wait, I wasn't thinking, there is an environment variable already built-in for that:
```
FASTLANE_USER
```
Could we use that in combination with a default value somehow?

**Comment by PaulTaykalo** [comment]
So what's your idea?
To put `FASTLANE_USER` into the `apple_id` ?
Or to set up `FASTLANE_USER` environment variable in Appfile?
Or ... some another option?

**Comment by PaulTaykalo** [comment]
The thing is if we'll set up `FASTLANE_USER` it'll override every other option anyway
The original idea was to set up apple id per application/ per user. So theoretically you can have different apple Ids for different applications etc.

**Comment by PaulTaykalo** [comment]
@KrauseFx

**Comment by KrauseFx** [comment]
Hey @PaulTaykalo, so let's remove the first approach where you tell the user to update the Appfile, and instead have the following as the first approach: 

You can set the username using the `FASTLANE_USER` environment variable, and have the default one defined in the `Appfile`

**Comment by PaulTaykalo** [comment]
@KrauseFx up

**Comment by KrauseFx** [comment]
One last thing, could you rebase from master to make the tests pass? Or submit a new PR for this 👍

**Comment by KrauseFx** [comment]
Hey @PaulTaykalo, so I rewrote some parts of it, and updated the code samples, and also added the PR to docs.fastlane.tools https://github.com/fastlane/docs/pull/115
Do you want to update this PR to match the same content? 👍

**Comment by KrauseFx** [comment]
Please also make sure the tests are passing by pulling the latest changes from master, otherwise we can't merge

**Comment by KrauseFx** [comment]
I'll close this PR for now, thanks for your contribution 👍

**Pull request closed without merging**